examples/vue/components/input.vue

```diff
@@ -13,10 +13,22 @@
       <text>oninput: {{txtInput}}</text>
       <text>onchange: {{txtChange}}</text>
     </panel>
+    <panel title="test input enter key type" type="primary">
+      <input
+        type="text"
+        placeholder="search ..."
+        class="input"
+        autofocus="true"
+        return-key-type="search"
+        @return="onreturn"
+      />
+      <text>enter key type: {{returnType}}</text>
+      <text>Action: {{msg}}</text>
+    </panel>
   </scroller>
 </template>
 
-<style scoped>
+<style>
```

Comment:
the style tag need to be scoped for weex. Otherwise some feature will not working for weex, some even to cause error.